src/Desktop.Analyzers/Core/Helpers/SecurityDiagnosticHelpers.cs

```diff
@@ -164,6 +165,30 @@ public static int HasXmlReaderParameter(IMethodSymbol method, CompilationSecurit
             return HasSpecifiedParameter(method, xmlTypes, IsXmlReaderType);
         }
 
+        public static bool IsExpressionEqualsNull(IOperation operation)
+        {
+            ILiteralExpression literal = operation as ILiteralExpression;
+            return literal != null && literal.ConstantValue.HasValue && literal.ConstantValue.Value == null; 
+        }
+
+        public static bool IsExpressionEqualsDtdProcessingParse(IOperation operation)
+        {
+            IFieldReferenceExpression enumRef = operation as IFieldReferenceExpression;
+            return enumRef != null && enumRef.ConstantValue.HasValue && (int)enumRef.ConstantValue.Value == 2; // DtdProcessing.Parse
```

Comment:
Yes, I have test cases. The reason I put 2 here is because in the current .NET CLR verion, this enum value is gone. To check it for other versions, a const will do the job.